classes/XLite/Module/Heartland/Securesubmit/Model/Payment/Securesubmit.php

```diff
@@ -294,16 +300,14 @@ protected function includeSecuresubmitLibrary()
                     ->findOneBy(array('service_name' => 'Securesubmit'));
                 $key = $method->getSetting('secretKey' . $suffix);
             }
-
-            $heartlandConfig = new \HpsServicesConfig();
-            $heartlandConfig->secretApiKey = $key;
-
-            $heartlandConfig->versionNumber = '1514';
-            $heartlandConfig->developerId = '002914';
-
-            $this->chargeService = new \HpsCreditService($heartlandConfig);
-
+        
+            $config = new ServicesConfig();
+            $config->secretApiKey = $key;
+            //$env = $config->environment;
+            $config->serviceUrl = 'https://cert.api2.heartlandportico.com'; 
```

Comment:
This will only hit sandbox/CERT. We need to update to support both sandbox and production.